hr_contract_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "HR Contract Operating Unit",
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
```

Comment:
@ageficent URL should be https://github.com/OCA/operating-unit

Reply:
Done :+1: